pydoctor/test/test_epydoc2stan.py

```diff
@@ -356,7 +356,21 @@ def __init__(self, p):
     assert captured == '<test>:5: Documented parameter "q" does not exist\n'
 
 
-def test_func_missing_exception_type(capsys: CapSys) -> None:
+def test_func_raise_linked() -> None:
+    """Raise fields are formatted by linking the exception type."""
+    mod = fromText('''
+    class SpanishInquisition(Exception):
+        pass
+    def f():
+        """
+        @raise SpanishInquisition: If something unexpected happens.
+        """
+    ''', modname='test')
+    html = docstring2html(mod.contents['f']).split('\n')
+    assert '<a href="test.SpanishInquisition.html">SpanishInquisition</a>' in html
+
+
+def test_func_raise_missing_exception_type(capsys: CapSys) -> None:
     """Raise fields must include the exception type."""
```

Comment:
I want to avoid `@raise` since that's epytext syntax, while we support other documentation formats as well (RST released and @tristanlatr has an experimental branch with Google/Numpy support).

The `test_func_raise_linked` test does check `@raise` with a type, but it doesn't verify that no warning is issued in that case, so I think there is some value in having both a correct and incorrect use of the "raise" field in this test.

I will update the docstring to mention the fact that the HTML is now checked as well.